redfish-core/lib/network_protocol.hpp

```diff
@@ -97,17 +108,19 @@ void getEthernetIfaceData(CallbackFunc&& callback)
             const boost::system::error_code errorCode,
             const dbus::utility::ManagedObjectType& dbusData) {
         std::vector<std::string> ntpServers;
+        std::vector<std::string> dynamicNtpServers;
         std::vector<std::string> domainNames;
 
         if (errorCode)
         {
-            callback(false, ntpServers, domainNames);
+            callback(false, ntpServers, dynamicNtpServers, domainNames);
             return;
         }
 
-        extractNTPServersAndDomainNamesData(dbusData, ntpServers, domainNames);
+        extractNTPServersAndDomainNamesData(dbusData, ntpServers,
+                                            dynamicNtpServers, domainNames);
 
-        callback(true, ntpServers, domainNames);
+        callback(true, ntpServers, dynamicNtpServers, domainNames);
     },
         "xyz.openbmc_project.Network", "/xyz/openbmc_project/network",
         "org.freedesktop.DBus.ObjectManager", "GetManagedObjects");
```

Comment:
It may be better to do the cherry-pick from [gerrit](https://gerrit.openbmc.org/c/openbmc/bmcweb/+/71026) (instead of from 1060) which will be more conformant to the expected coding style.

Reply:
For 1110 we can use the upstream commit as it is. Changing 1060 would add more code changes at this time of GA.

Reply:
I don't mean to change 1060.
I meant - This PR follows 1060 coding style, not upstream coding style.  It may be better to use upstream coding style (i.e. using gerrit code) into 1110.